docs/.vuepress/config.js

```diff
@@ -12,13 +12,16 @@ module.exports = {
   chainMarkdown (config) {
     config.plugin('add-metadata')
       .use(markdown => {
-        markdown.core.ruler.push('add-date', state => {
+        markdown.core.ruler.push('add-metadata', state => {
```

Comment:
Everywhere else "meta data" is now used as two words. So do the same in this chain?